extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -113,6 +113,24 @@ export class AmpSelector extends AMP.BaseElement {
       this.element.addEventListener('click', this.clickHandler_.bind(this));
       this.element.addEventListener('keydown', this.keyDownHandler_.bind(this));
     }
+
+    this.registerAction('selectUp', invocation => {
```

Comment:
Bikeshed naming - should the attribute be changed to `skip` and then the dev can pass in negative or positive values to the function `select`?